common-content/en/module/js3/template-html/index.md

```diff
@@ -1,11 +1,12 @@
 +++
-title = '🍱 Simplifying element creation'
+title = '🍱 Simplifying element creation with <template> tags'
```

Comment:
again here -- titles are too long. 

Really don't want titles longer than 35ch for readiablity. But 55, 60ch will get chomped in search results too!

Reply:
Similarly here, having two consecutive blocks in the overview which are both titled "Simplifying element creation" seems unhelpful...

Reply:
How about 

Creating elements with <template> ?